### ISSUE TITLE
If endpoint has a record field in the output type OpenAPI fails to start

### ISSUE THREAD

**Issue by kirdk** [RobinTail/express-zod-api#92]
If I'm trying to create an endpoint with an output that contains a field with type z.record() or z.any(), the OpenAPI builder fails with the message:
```
error: Cannot convert undefined or null to object
error: TypeError: Cannot convert undefined or null to object
```

**Comment by RobinTail** [comment]
Thank you @kirdk .

`z.record()` — confirmed, this will be fixed.
`z.any()` — should throw another error: `Zod type ZodAny is unsupported`. Did not yet figure out how to describe `any` in Swagger.

**Comment by RobinTail** [comment]
Both issues fixed in v2.1.1, @kirdk

**Comment by kirdk** [comment]
> Both issues fixed in v2.1.1, @kirdk

Thank you @RobinTail
